internal/wireguard/netlink_integration_test.go

```diff
@@ -72,7 +72,7 @@ func Test_netlink_Wireguard_addRule(t *testing.T) {
 	rulePriority := 10000
 	const firewallMark = 999
 
-	cleanup, err := wg.addRule(rulePriority, firewallMark)
+	cleanup, err := wg.addRule4(rulePriority, firewallMark)
```

Comment:
Please rename the test from `Test_netlink_Wireguard_addRule` to `Test_netlink_Wireguard_addRule4`